sktime/classification/dictionary_based/tests/test_boss.py

```diff
@@ -35,6 +35,19 @@ def test_individual_boss_on_gunpoint():
     testing.assert_array_equal(probas, individual_boss_gunpoint_probas)
 
 
+def test_boss_on_power_demand():
```

Comment:
I appreciate this new test and the one below for WEASEL too, I'm only a bit worried about limited run times on free CI services, if there's anything you can do to make them quicker that would be great e.g. using a subset of the data, fixing certain parameters, but haven't checked yet how much time they actually need

Reply:
Oh, sure. I was not aware of this issue. I noticed a problem while editing the jupyter-notebook, which was not covered by tests. Thus, I added a test. 

The additional BOSS test takes 9s. I am fine to uncomment the test again - what do you think?

The two tests in WEASEL take only 4.5s, so they are quite fast. Is that ok?

Reply:
Yeah that seems fine!

Reply:
I have reduced the runtime by using a subset of the test data, too.